rosa/cli.py

```diff
@@ -72,13 +76,20 @@ def change_home_environment():
     os.environ["HOME"] = current_home
 
 
-def is_logged_in(aws_region=None, allowed_commands=None):
+def is_logged_in(env, aws_region=None, allowed_commands=None):
     _allowed_commands = allowed_commands or parse_help()
+
     try:
         res = build_execute_command(command="whoami", aws_region=aws_region, allowed_commands=_allowed_commands)
-        return "User is not logged in to OCM" not in res["err"]
-    except CommandExecuteError:
-        return False
+
+        logged_in_env = res["out"].get("OCM API")
```

Comment:
```suggestion
        ocm_env = res["out"].get("OCM API")
```

Reply:
changed to `logged_in_ocm_env` so it is clear that this is the logged in env